test/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,53 @@
+import core.basesyntax.service.FileReader;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import core.basesyntax.service.impl.FileReaderImpl;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String FILE_EXIST = "test/resources/input.csv";
+    private static final String FILE_NOT_EXIST = "wrong/file.csv";
+    private static FileReader fileReaderTest;
+
+    @BeforeClass
+    public static void setUp() {
+        fileReaderTest = new FileReaderImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readWrongFile_Not_Ok() {
+        fileReaderTest.readFromFile(FILE_NOT_EXIST);
+    }
+
+    @Test
+    public void readFile_Ok() throws IOException {
```

Comment:
Lets remove throws (who catches this?) from method referance and use try-catch instad?